app/src/main/java/org/thoughtcrime/securesms/conversation/v2/Util.kt

```diff
@@ -105,7 +97,7 @@ object Util {
     }
 
     fun isEmpty(charSequence: CharSequence?): Boolean {
-        return charSequence == null || charSequence.length == 0
+        return charSequence.isNullOrEmpty()
```

Comment:
Can't we simply remove the Util function and just rely on `isNullOrEmpty` at the call site? A lot of these Util were created a long time ago and have direct replacements, like in this case.

Reply:
I guess...

Git is lying to you (and me) by saying I wrote these **isEmpty** method in `Utill.kt` - the only one I wrote was ` fun usingLeftToRightLanguage(context: Context): Boolean` the rest are legacy methods I had nothing to do with.

Removing the four **isEmpty** methods still allows the code to build, so they're unnecessary & I've deleted them.